app/javascript/dashboard/i18n/locale/en/report.json

```diff
@@ -508,15 +508,32 @@
   },
   "SLA_REPORTS": {
     "HEADER": "SLA Reports",
+    "NO_RECORDS": "SLA applied conversations are not available.",
+    "LOADING": "Loading SLA data...",
     "METRICS": {
       "HIT_RATE": {
         "LABEL": "Hit Rate",
         "TOOLTIP": "Percentage of SLAs created were completed successfully"
       },
       "NO_OF_BREACHES": {
-        "LABEL": "Number of Breaches",
-        "TOOLTIP": "The total SLA breaches in a certain period."
+        "LABEL": "Number of Misses",
+        "TOOLTIP": "Total SLA misses in a certain period"
+      },
+      "NO_OF_CONVERSATIONS": {
+        "LABEL": "Number of Conversations",
+        "TOOLTIP": "Total number of conversations with SLA"
       }
+    },
+    "TABLE": {
+      "HEADER": {
+        "POLICY_BREACHED": "Policy",
+        "CONVERSATION": "Conversation",
+        "AGENT": "Agent"
+      },
+      "COLUMN": {
+        "CONVERSATION": "<span class='text-slate-700 dark:text-slate-200'>#{conversationId}</span> with <span class='text-slate-700 dark:text-slate-200 capitalize truncate'>{contactName}</span>"
```

Comment:
I don't think this it is a good idea to have markup and classes here. Let's try to see if we can get this done any other way? Perhaps changing the copy itself can help

Reply:
Yeah. It will not work.